Added a currentVersion prop to the Tracks event for the JetpackBanner

robertf4 opened the pull request:
The Tracks event for the `JetpackBanner` was not logging the version, making it hard to know which version of the client was sending users to the redirect links. This adds a `currentVersion` prop to fix that.

#### Changes proposed in this Pull Request:
* Adds a `currentVersion` prop to the `JetpackBanner` Tracks event.

#### Is this a new feature or does it add/remove features to an existing part of Jetpack?
* This is part of the At-a-Glance update project.

#### Testing instructions:
* Go to the At-a-Glance dashboard on a Jetpack free plan.
* Enter `localStorage.setItem('debug', 'dops:analytics')` in the console and refresh the page in order to see Tracks events.
* Set the console to preserve log.
* Click the Upgrade button on the Scan card.
* In the console, verify that a `jetpack_wpa_click` event was fired with a `currentVersion` prop correctly set.

#### Proposed changelog entry for your changes:
* None needed.

robertf4 committed:
Added a currentVersion prop to the Tracks event for the JetpackBanner

lucasartoni commented:
I get the correct parameter on `jetpack_wpa_click` but I don't see it on `jetpack_wpa_page_view`

lucasartoni commented:
The parameter should be `current_version` instead of `currentVersion`. @robertf4

robertf4 committed:
Renamed currentVersion to current_version

robertf4 committed:
Added current_version as a prop to the page view Tracks event

robertf4 commented:
> I get the correct parameter on jetpack_wpa_click but I don't see it on jetpack_wpa_page_view

I've updated the page view to log `current_version` now.

> The property should be current_version instead of currentVersion.

Changed.

lucasartoni commented:
It works as expected on `jetpack_wpa_page_view` and `jetpack_wpa_click` 👍

zinigor reviewed (approved):
As we have discussed with @robertf4 in a private discussion, adding a redux wrapper to the banner component seems to be the easier way of doing this for now, until we find a way to have the tracks library access the state tree directly for things like current version, etc.

jeherve commented:
Cherry-picked to `branch-7.5` in 6db18fa01e8aa346342c11352ab759d4618989f7
